rfc/002-extension-content-configuration-processing.md

```diff
@@ -36,6 +38,9 @@ metadata:
   name: example-extension
   namespace: default
   generation: 1
+  labels:
+    extensionclass.openmfp.io: extension-class-sample1
+    extensionNamespace: default
```

Comment:
```suggestion
```

Reply:
@aaronschweig Do you mean to make the label optional?